examples/fscloud/variables.tf

```diff
@@ -0,0 +1,196 @@
+  }
+}
+
+variable "region" {
+  type        = string
+  description = "Region where resources are created"
+  default     = "us-south"
+}
+
+variable "resource_group" {
+  type        = string
+  description = "An existing resource group name to use for this example, if unset a new resource group will be created"
+  default     = null
+}
+
+variable "resource_tags" {
+  type        = list(string)
+  description = "Optional list of tags to be added to created resources"
+  default     = []
+}
+
+variable "ocp_version" {
+  type        = string
+  description = "Version of the OCP cluster to provision"
+  default     = null
+}
+
+variable "primary_existing_hpcs_instance_guid" {
+  description = "The GUID of the Hyper Protect Crypto service in which the key specified in var.hpcs_key_crn is coming from, used by the COS instance"
+  type        = string
+}
+
+variable "secondary_existing_hpcs_instance_guid" {
+  description = "The GUID of the Hyper Protect Crypto service in which the key specified in var.hpcs_key_crn is coming from, used by the COS instance"
+  type        = string
+}
+
+variable "primary_hpcs_key_crn" {
+  description = "CRN of the Hyper Protect Crypto service to use to encrypt the data in the COS Bucket"
+  type        = string
+}
+
+variable "secondary_hpcs_key_crn" {
+  description = "CRN of the Hyper Protect Crypto service to use to encrypt the data in the COS Bucket"
+  type        = string
+}
+
+variable "hpcs_instance_guid" {
+  type        = string
+  description = "The GUID of the Hyper Protect Crypto service to provision the encryption keys"
+
+}
+
+variable "hpcs_key_crn_cluster" {
+  description = "CRN of the Hyper Protect Crypto service to use to encrypt the cluster boot volume"
+  type        = string
+}
+
+variable "hpcs_key_crn_worker_pool" {
+  description = "CRN of the Hyper Protect Crypto service to use to encrypt the worker pool boot volumes"
+  type        = string
+}
+variable "existing_at_instance_crn" {
+  type        = string
+  description = "Optionally pass an existing activity tracker instance CRN to use in the example. If not passed, a new instance will be provisioned"
+  default     = null
+}
+
+variable "worker_pools" {
+  type = list(object({
+    subnet_prefix     = string
+    pool_name         = string
+    machine_type      = string
+    workers_per_zone  = number
+    resource_group_id = optional(string)
+    labels            = optional(map(string))
+    boot_volume_encryption_kms_config = optional(object({
+      crk             = string
+      kms_instance_id = string
+      kms_account_id  = optional(string)
+    }))
+  }))
+  description = "List of worker pools."
+  default     = []
+}
+
+##############################################################################
+# VPC variables
+##############################################################################
+
+variable "vpc_name" {
+  type        = string
+  description = "Name of the VPC"
+  default     = "management"
+}
```

Comment:
Do vpc's have to have a unique name? If so, I suggest removing this variable, and hard code the vpc name in the main.tf of the example using the prefix variable